src/nouns-coasters/utils/InflateLib.sol

```diff
@@ -0,0 +1,799 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.0 <0.9.0;
+
+//
+// inflate content script:
+// var pako = require('pako')
+// var deflate = (str) => [str.length,Buffer.from(pako.deflateRaw(Buffer.from(str, 'utf-8'), {level: 9})).toString('hex')]
+//
+
+/// @notice Based on https://github.com/madler/zlib/blob/master/contrib/puff
+library InflateLib {
```

Comment:
Not sure of the read gas limits that NFT marketplaces impose these days, but [this inflate lib](https://github.com/nounsDAO/nouns-monorepo/blob/master/packages/nouns-contracts/contracts/libs/Inflate.sol) will reduce `tokenURI` gas by roughly 50%. Nouns has used this implementation in prod for roughly a year.